api/v6/report_server.thrift

```diff
@@ -25,6 +25,22 @@ struct RunData {
 }
 typedef list<RunData> RunDataList
 
+struct RunHistoryData {
+  1: i64       runId,    // Unique id of the run.
+  2: string    runName,  // Name of the run.
+  3: string    tag,      // Tag of the report.
```

Comment:
Shouldn't this be the tag of the run?
Will we support only one tag for each history data entry?

Reply:
- A version tag is uniquely identifying a store. Actually, the last tag at the run history data is the tag of the run.
- We support only one tag for each history data as I know.